resources/kcp/charts/kyma-environment-broker/templates/subaccount-cleanup-job.yaml

```diff
@@ -268,7 +268,7 @@ spec:
                 - name: APP_BROKER_SCOPE
                   value: {{ .Values.kebClient.scope }}
               command:
-                - "/bin/accountcleanup"
+                - "/bin/main"
```

Comment:
Will break the job. You changes should not affect jobs other than environment cleanup (according to what you said).